src/manifest.firefox.json

```diff
@@ -1,4 +1,5 @@
 {
+	"author": "Jacob Tender",
```

Comment:
firefox doesn't like an object here, so overriding for firefox to just author name
<img width="631" height="81" alt="image" src="https://github.com/user-attachments/assets/915e43f2-e9c9-4fee-a630-66d7ad421e11" />